test/test_jsonrpc_client.ml

```diff
@@ -59,6 +59,9 @@ module Input_json_object = Generic.Make (struct
 		(* A file containing a partial JSON object. *)
 		"short_call.json", Left Parse_error;
 
+		(* A file containing a JSON object, plus some more characters at the end. *)
+		"good_call_plus.json", Right good_call;
```

Comment:
Did you need to `git add` this file?

Reply:
this is an existing file, which is not removed the last time we remove this case.